pymc3/sampling.py

```diff
@@ -297,6 +298,9 @@ def sample(draws=500, step=None, init='auto', n_init=200000, start=None,
     compute_convergence_checks : bool, default=True
         Whether to compute sampler statistics like gelman-rubin and
         effective_n.
+    use_mmap : bool, default=True
+        Whether to use joblib's memory mapping to share numpy arrays when
+        sampling across multiple cores.
```

Comment:
I think the default should be `False`. Right now passing arrays using mmap seems mostly broken, right? Theano just won't read those.

Reply:
I was trying to maintain backwards compatibility by default, which is to use mmap unless I'm mistaken, but I don't feel terribly strongly either way.

Reply:
I think "backward compatibility" would mean: raise an error and refuse to sample. I think in this case we can live with that. :-)